bower.json

```diff
@@ -24,6 +24,7 @@
   },
   "devDependencies": {
     "test-fixture": "polymerelements/test-fixture#^1.0.0",
+    "promise-polyfill": "polymerlabs/promise-polyfill#^1.0.0",
```

Comment:
I feel ominous dread knowing that we are using this in more places now. Coupled with delight. An interesting combination :candy: :skull: